### PR TITLE
Update pipeline quickstart content

### PR DESCRIPTION
**Pull request by karthikjeeyar**
# Changes
Updates the  ConsoleQuickStart CR 

Updated the content of the pipeline quickstarts to match the current flow in OCP

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml [hunk 1]
```diff
@@ -53,26 +53,28 @@ spec:
       success: You just located the associated Pipeline! Next, we'll start and explore
         your Pipeline.
       failed: Check your work to locate the associated pipeline.
-  - title: Starting your pipeline
+  - title: Exploring your pipeline run
     description: |-
-      ### You’ve just explored the topology of your application and seen it’s related resources.  Now let’s start your pipeline.
+      ### You’ve just explored the topology of your application and seen it’s related resources.  Now let’s explore your pipeline run.
       Notice the Pipeline Runs section of the Side Panel
       1. The first row shows the Pipeline associated with the application and an action button to **Start Last Run**.
```
Comment (serenamarie125) [inline cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml:60]: @karthikjeeyar let's just use:
1. The first row shows the Pipeline associated with the application.

File: cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml [hunk 2]
```diff
@@ -53,26 +53,28 @@ spec:
...
-      2. Click on the Pipeline link, which should bring you to the **Pipeline Details** page.
-      3. From the Action menu, click on **Start** to start your Pipeline.
+      2. The second row shows the recently created pipeline run.
```
Comment (serenamarie125) [inline cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml:61]: 2. The second row shows the recently created Pipeline Run, its status and a link to view its logs.

File: cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml [hunk 3]
```diff
@@ -53,26 +53,28 @@ spec:
...
+      3. Click on the pipeline run link.
     review:
```
Comment (serenamarie125) [inline cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml:63]: 3. Click on the Pipeline Run link to view the **Pipeline Run Details** page.

File: cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml [hunk 4]
```diff
@@ -53,26 +53,28 @@ spec:
...
-        #### Momentarily, you should be brought to the **Pipeline Run Details** page. To verify that your pipeline has started,
+        #### Momentarily, you should be brought to the **Pipeline Run Details** page.
         1. Note that the **Pipeline Runs Details** section shows a visualization of the pipeline run status and the tasks in the pipeline.
         2. Hovering over a task shows a tooltip with the details of the associated steps.
-        3. Click on the **Logs** tab to watch the progress of your pipeline run.
-        4. When the pipeline run is complete, you will see a **Succeeded** badge on the page title.
+        3. Click on the **Task Runs** tab to view all the task runs associated with the pipeline run.
+        4. Click on the **Logs** tab to watch the progress of your pipeline run.
+        5. Click on the **Events** tab to watch the event streams of your pipeline run.
+        6. When the pipeline run is complete, you will see a **Succeeded** badge on the page title.
```
Comment (serenamarie125) [inline cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml:71]: 6. When the pipeline run is complete, the **Running** badge on the page title will change to **Succeeded**.

**Review by serenamarie125 — CHANGES_REQUESTED** [review]

**Comment by serenamarie125** [comment]
@karthikjeeyar @andrewballantyne i've added some suggestions, feel free to take what you think makes sense!

File: cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml (hunk 4 above)
Comment (andrewballantyne) [reply cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml:71]: `Pipeline Run` capitalized

File: cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml (hunk 3 above)
Comment (andrewballantyne) [reply cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml:63]: `Pipeline Run details` to match the view capitalization

**Review by andrewballantyne — COMMENTED** [review]
General comment about how we have started doing i18n in OpenShift. 

- Dehumanizing Kinds is probably the most notable one (eg `Pipeline Run` to `PipelineRun`)
- We are not currently support translations for this @serenamarie125, is that a desire? Not sure how one goes about doing that with Quick Starts, handing off to the translation team I suspect

Not sure we _need_ to do those for the quick starts today, but it's worth noting.

**Comment by serenamarie125** [comment]
> * We are not currently support translations for this @serenamarie125, is that a desire? Not sure how one goes about doing that with Quick Starts, handing off to the translation team I suspect
> 
> Not sure we _need_ to do those for the quick starts today, but it's worth noting.

Agreed! I have mentioned to Christian & Ali, will bring it up again as I don't remember the details of their responses!

**Comment by karthikjeeyar** [comment]
Updated the PR with suggested changes.
cc: @vdemeester

**Review by vdemeester — APPROVED** [review]
/meow
/cc @savitaashture @piyush-garg

**Review by savitaashture — COMMENTED** [review]
/lgtm

**Pull request merged**